docs/doxygen-user/multi-user/installPostgres.dox

```diff
@@ -0,0 +1,119 @@
+/*! \page install_postgresql_page Install and Configure PostgreSQL
+
+
+\section install_post_over Overview
+
+In a multi-user case, a central PostgreSQL database server is used instead of the embedded SQLite databases.  
+
+A new database will be created for each case and the database will be stored on the local drive.  
```

Comment:
With the introduction of the Central Repository it may no longer be a good idea to store databases in the default location (system drive). We might want to redo this section to encourage users to choose a location where databases will be stored.